CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+Thanks for considering to contribute to this project! A few guidelines for
+people who want to contribute their code to this software are documented at
+this project's Wiki. If you're not sure where to start head on to [this wiki page](https://github.com/commons-app/apps-android-commons/wiki/Volunteers-welcome!).
+
+Here's a jist of the guidelines,
+
+# Make separate commits for logically separate changes
+
+# Describe your changes well in the commit message
+
+The first line of the commit message should be a short description of what has
+changed. It is also good to prefix the first line with "area: " where the "area"
+is a filename or identifier for the general area of the code being modified.
+The body should provide a meaningful commit message.
+
+# Write tests for your code
```

Comment:
How about adding "if possible"?
Sometimes a UI change can not really be tested via unit tests.